api/src/paths/xlsx/validate.ts

```diff
@@ -72,6 +76,50 @@ function prepXLSX(): RequestHandler {
   };
 }
 
+export function getValidationSchema(): RequestHandler {
```

Comment:
once restored for dwc, delete from here

Reply:
we need it in xlsx but not in dwc

Reply:
cause for xlsx we want to get the validation schema but for dwc we dont as of now

Reply:
the old workflow was in dwc, and called by xlsx.  Either way, we need to make sure they're not able to upload dwc for the time being.

Reply:
if they are unable to upload dwc then all the work ive done in this ticket is useless and cannot be tested...

Reply:
can we leave it this way until the transformation piece is in place

Reply:
right .... I see what you mean.  Can we have the client test in openshift?

@NickPhura which way should we go?

Reply:
I'd say leave it for now and then when transformation is done and in dev, we just change this to not allow dwc and it will still work since the xlsx can be transformed to dwc and this functionality can still be called